Bottom: fc2f250ea63c70dc439366b9f5fe1ca96637e8e0
Top:    368e0f4e8ad79fc12862755bf01506cad46701fd
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 19:16:14 +0000

Comment object3d_draw_debug() call during expose


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 85bffb4..37340d4 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -2707,7 +2707,7 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
   glDisable (GL_LIGHTING);
 
   draw_crosshair (Output.fgGC, priv);
-  object3d_draw_debug ();
+//  object3d_draw_debug ();
 
   hidgl_flush_triangles (priv->hidgl);
